src/ILLink.Tasks/LinkTask.cs

```diff
@@ -42,8 +42,7 @@ public class ILLink : ToolTask
 		///   is subject to change. Currently these get passed to
 		///   illink with "-a", which roots the entry point for
 		///   executables, and everything for libraries. To control
-		///   the linker more explicitly, either pass descriptor
-		///   files, or pass extra arguments for illink.
+		///   the behaviour explicitly, set RootsMode metadata.
```

Comment:
May I suggest the name "RootMode", singular? Also the description doesn't match the GetMetadata call.

Reply:
Yep, happy to take it (went back and forth on this)